### ISSUE TITLE
Getting Error on import to setup_js_tag_helper

### ISSUE THREAD

**Issue by stevenjorgensen67** [sphinx-doc/sphinx#11633]
### Describe the bug

ImportError: cannot import name 'setup_js_tag_helper' from 'sphinx.builders.html'

### How to Reproduce

sphinx-build -M html "source" "doc_build"  
Running Sphinx v7.2.2

Exception occurred:
  File ".venv/lib/python3.10/site-packages/notfound/extension.py", line 337, in setup
    from sphinx.builders.html import setup_js_tag_helper
ImportError: cannot import name 'setup_js_tag_helper' from 'sphinx.builders.html' (.venv/lib/python3.10/site-packages/sphinx/builders/html/__init__.py)
The full traceback has been saved in /tmp/sphinx-err-_5rxd6sg.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!

### Environment Information

```text
sphinx
sphinx-notfound-page
recommonmark
plantweb
sphinx-tabs
sphinx_rtd_theme
sphinx-autobuild
graphviz
sphinxcontrib-xlsxtable==1.0.0
```

### Additional context

# Traceback:
Traceback (most recent call last):
  File ".venv/lib/python3.10/site-packages/sphinx/cmd/build.py", line 293, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File ".venv/lib/python3.10/site-packages/sphinx/application.py", line 233, in __init__
    self.setup_extension(extension)
  File ".venv/lib/python3.10/site-packages/sphinx/application.py", line 406, in setup_extension
    self.registry.load_extension(self, extname)
  File ".venv/lib/python3.10/site-packages/sphinx/registry.py", line 460, in load_extension
    metadata = setup(app)
  File ".venv/lib/python3.10/site-packages/notfound/extension.py", line 337, in setup
    from sphinx.builders.html import setup_js_tag_helper
ImportError: cannot import name 'setup_js_tag_helper' from 'sphinx.builders.html' (.venv/lib/python3.10/site-packages/sphinx/builders/html/__init__.py)

**Comment by mattatark** [comment]
Yep, same issue for me.

## Traceback

```
Running Sphinx v7.2.2

Traceback (most recent call last):
  File "/root/pysparkplug/.cache/nox/docs/lib/python3.10/site-packages/sphinx/cmd/build.py", line 293, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/root/pysparkplug/.cache/nox/docs/lib/python3.10/site-packages/sphinx/application.py", line 233, in __init__
    self.setup_extension(extension)
  File "/root/pysparkplug/.cache/nox/docs/lib/python3.10/site-packages/sphinx/application.py", line 406, in setup_extension
    self.registry.load_extension(self, extname)
  File "/root/pysparkplug/.cache/nox/docs/lib/python3.10/site-packages/sphinx/registry.py", line 460, in load_extension
    metadata = setup(app)
  File "/root/pysparkplug/.cache/nox/docs/lib/python3.10/site-packages/notfound/extension.py", line 337, in setup
    from sphinx.builders.html import setup_js_tag_helper
ImportError: cannot import name 'setup_js_tag_helper' from 'sphinx.builders.html' (/root/pysparkplug/.cache/nox/docs/lib/python3.10/site-packages/sphinx/builders/html/__init__.py)

Exception occurred:
  File "/root/pysparkplug/.cache/nox/docs/lib/python3.10/site-packages/notfound/extension.py", line 337, in setup
    from sphinx.builders.html import setup_js_tag_helper
ImportError: cannot import name 'setup_js_tag_helper' from 'sphinx.builders.html' (/root/pysparkplug/.cache/nox/docs/lib/python3.10/site-packages/sphinx/builders/html/__init__.py)
```

## Environment

### Python version

Python 3.10.12

### Installed Packages

```
alabaster==0.7.13
    # via sphinx
babel==2.12.1
    # via sphinx
backports-strenum==1.2.4 ; python_version < "3.11"
    # via -r /root/pysparkplug/requirements/requirements.txt
beautifulsoup4==4.12.2
    # via furo
certifi==2023.7.22
    # via requests
charset-normalizer==3.2.0
    # via requests
contourpy==1.1.0
    # via matplotlib
cycler==0.11.0
    # via matplotlib
docutils==0.20.1
    # via
    #   myst-parser
    #   sphinx
fonttools==4.42.1
    # via matplotlib
furo==2023.8.19
    # via -r /root/pysparkplug/requirements/docs.in
idna==3.4
    # via requests
imagesize==1.4.1
    # via sphinx
jinja2==3.1.2
    # via
    #   myst-parser
    #   sphinx
kiwisolver==1.4.4
    # via matplotlib
markdown-it-py==3.0.0
    # via
    #   mdit-py-plugins
    #   myst-parser
markupsafe==2.1.3
    # via jinja2
matplotlib==3.7.2
    # via sphinxext-opengraph
mdit-py-plugins==0.4.0
    # via myst-parser
mdurl==0.1.2
    # via markdown-it-py
myst-parser==2.0.0
    # via -r /root/pysparkplug/requirements/docs.in
numpy==1.25.2
    # via
    #   contourpy
    #   matplotlib
packaging==23.1
    # via
    #   -r /root/pysparkplug/requirements/docs.in
    #   matplotlib
    #   sphinx
paho-mqtt==1.6.1
    # via -r /root/pysparkplug/requirements/requirements.txt
pillow==10.0.0
    # via matplotlib
protobuf==4.24.1
    # via -r /root/pysparkplug/requirements/requirements.txt
pygments==2.16.1
    # via
    #   furo
    #   sphinx
pyparsing==3.0.9
    # via matplotlib
python-dateutil==2.8.2
    # via matplotlib
pyyaml==6.0.1
    # via myst-parser
requests==2.31.0
    # via sphinx
six==1.16.0
    # via python-dateutil
snowballstemmer==2.2.0
    # via sphinx
soupsieve==2.4.1
    # via beautifulsoup4
sphinx==7.2.2
    # via
    #   -r /root/pysparkplug/requirements/docs.in
    #   furo
    #   myst-parser
    #   sphinx-basic-ng
    #   sphinx-copybutton
    #   sphinx-notfound-page
    #   sphinxcontrib-applehelp
    #   sphinxcontrib-devhelp
    #   sphinxcontrib-htmlhelp
    #   sphinxcontrib-qthelp
    #   sphinxcontrib-serializinghtml
    #   sphinxext-opengraph
sphinx-basic-ng==1.0.0b2
    # via furo
sphinx-copybutton==0.5.2
    # via -r /root/pysparkplug/requirements/docs.in
sphinx-notfound-page==0.8.3
    # via -r /root/pysparkplug/requirements/docs.in
sphinxcontrib-applehelp==1.0.7
    # via sphinx
sphinxcontrib-devhelp==1.0.5
    # via sphinx
sphinxcontrib-htmlhelp==2.0.4
    # via sphinx
sphinxcontrib-jsmath==1.0.1
    # via sphinx
sphinxcontrib-qthelp==1.0.6
    # via sphinx
sphinxcontrib-serializinghtml==1.1.9
    # via sphinx
sphinxext-opengraph==0.8.2
    # via -r /root/pysparkplug/requirements/docs.in
typing-extensions==4.7.1 ; python_version < "3.11"
    # via -r /root/pysparkplug/requirements/requirements.txt
urllib3==2.0.4
    # via requests
```

**Comment by humitos** [comment]
Hi! This is an issue on the `sphinx-notfound-page` extension: https://github.com/readthedocs/sphinx-notfound-page/issues/219

**Comment by stevenjorgensen67** [comment]
I did not see it until the release of Sphinx 7.2. Everything works on Sphinx 7.1

**Comment by mattatark** [comment]
Thanks @humitos ! Seems like we ought to close this ticket out, apologies for not catching that in the traceback.

**Comment by michaelweinold** [comment]
Issue now resolved with sphinx>7.2.X following the release of sphinx-notfound-page=1.0.0 on conda-forge: https://github.com/conda-forge/sphinx-notfound-page-feedstock/pull/5#event-10363420783
